pos_membership/static/src/xml/PartnerLine.xml

```diff
@@ -15,7 +15,7 @@
         <xpath expr="//button[hasclass('edit-partner-button')]" position="after">
             <span
                 t-if="props.partner.membership_state != 'none'"
-                class="membership-state-tag"
+                t-attf-class="membership-state-tag {{ props.partner.is_member ? 'membership-green' : 'membership-red' }}"
```

Comment:
this could also be written as (see [owl’s doc about dynamic class attributes](https://github.com/odoo/owl/blob/master/doc/reference/templates.md#dynamic-class-attribute)):
```suggestion
                class="membership-state-tag"
                t-att-class="{'membership-green': props.partner.is_member, 'membership-red': !props.partner.is_member}"
```
but it is more verbose and doesn't allow for an “else” clause, which forces the condition to be evaluated twice.

Reply:
@huguesdk I've just made an update with another proposal. What do you think?

Reply:
Juste to notice that there is a fix to do in [this module](https://github.com/coopiteasy/cie-custom/tree/16.0/lpcr_pos_membership), when this change will be validated.

Reply:
@huguesdk Done

Reply:
many thanks!